crates/cext/cbindgen.toml

```
@@ -13,14 +13,15 @@ after_includes = """
 #include "qiskit/complex.h"
 
 #define QISKIT_RELEASE_LEVEL_DEV 0xA // dev or "alpha"
+#define QISKIT_RELEASE_LEVEL_BETA 0xB // beta
 #define QISKIT_RELEASE_LEVEL_RC 0xC // release candidate
 #define QISKIT_RELEASE_LEVEL_FINAL 0xF // final release
 
 #define QISKIT_VERSION_MAJOR 2
 #define QISKIT_VERSION_MINOR 2
 #define QISKIT_VERSION_PATCH 0
-#define QISKIT_RELEASE_LEVEL QISKIT_RELEASE_LEVEL_DEV
-#define QISKIT_RELEASE_SERIAL 0  // for the final release, this should be 0
+#define QISKIT_RELEASE_LEVEL QISKIT_RELEASE_LEVEL_BETA
+#define QISKIT_RELEASE_SERIAL 1  // for the final release, this should be 0
 
 #define QISKIT_VERSION "2.2.0dev"
```

Should I change this line too?

---

Yes, this string should match the qiskit python version string, so I'd expect:

```suggestion
#define QISKIT_VERSION "2.2.0b1"
```